test/io/ferguson/json.chpl

```diff
@@ -64,12 +64,13 @@ var f = opentmp();
   var reader = f.reader();
 
   var r:MyRecord;
+  var got: bool;
 
-  var err:syserr;
-
-  var got = reader.readf("%~jt", r, error=err);
-
-  writeln("got is ", got, " error is ", errorToString(err));
+  try! {
+    got = reader.readf("%~jt", r);
+  } catch e: SystemError {
+    writeln("got is ", got, " error is ", errorToString(e.err));
```

Comment:
I think this test should print out "got is " whether or not there was an error. You'll need to adjust the .good file.